client/src/utils/dateUtil.js

```diff
@@ -0,0 +1,71 @@
+import moment from 'moment';
+
+export function weekUtil(currentDay, firstOfMonthDay, lastOfMonthDate, weeksInMonth, weeksLength){
```

Comment:
@blaketarter Updated the week and month functions so that the weekUtil could be used independently of the monthUtil. Let me know how it's looking now.